gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/engine/ui/SnappyRegionStatsCollectorFunction.java

```diff
@@ -55,6 +55,9 @@ public boolean hasResult() {
 
   @Override
   public void execute(FunctionContext context) {
+
+    com.pivotal.gemfirexd.internal.snappy.CallbackFactoryProvider.getClusterCallbacks().
```

Comment:
Also please make a note in the mvcc/column table update/delete story in jira to track modifying this number with the deleted rows bitset when the column/table update/delete comes.